keyboards/babyv/keymaps/melonbred/keymap.c

```diff
@@ -0,0 +1,62 @@
+/* Copyright 2020 Eye Oh! Designs
```

Comment:
```suggestion
/* Copyright 2020 melonbred
```

All the other licenses specify "melonbred", I don't think it matters much but I think a consistent attribution would be preferable.